addOns/help/src/main/javahelp/contents/releases/2.12.0.html

```diff
@@ -9,6 +9,42 @@
 <BODY>
 <H1>Release 2.12.0</H1>
 
+This is a bug fix and enhancement release, which now requires a minimum of Java 11. 
+<br><br>
+These release notes do not include all of the changes included in add-ons updated since 2.11.1.
+<br><br>
+Some of the more significant enhancements include:
+
+<H3>Network Add-On</H3>
+
+The core networking code has been replaced by a new add-on which means changes are no longer bound to core/stable releases.
+This add-on uses a modern network stack which will make it much easier to support modern protocols such as HTTP/2.
+<br>
+In addition the following features have been added:
+<ul>
+  <li>HTTPS pass-through
+  <li>Certificate validity period configuration 
+</ul>
+
+<H3>Spider Add-On</H3>
+
+To facilitate more frequent functional enhancements and bug fixes the core Spider has been moved to an add-on which means such changes are no longer bound to core/stable releases.
+Other add-ons which use Traditional Spider functionality have also been re-worked to support the Spider add-on, including: Quick Start, Form Handler, GraphQL, OpenAPI, SOAP, and the Automation Framework. 
+More details are given below.
+
+<H3>Multi-threaded Passive Scanner</H3>
+
+The passive scanner has been updated to use a configurable number of threads, by default 4.
+This has been shown to significantly reduce the time spent processing the passive scan queue.
+
+<h3>Dependency Updates</h3>
```

Comment:
We'll have to note the ones that are being removed (can do that later, it's tied to the changes of the Network add-on).

Reply:
We already have some mentioned below, we should merge these sections.

Reply:
But there's another section between and then `Changes in Bundled Libraries`.